src/components/button.tsx

```diff
@@ -2,9 +2,14 @@ import React, { forwardRef, type ElementRef, type ComponentProps } from "react";
 
 const defaultTag = "button";
 
+export type ButtonProps = ComponentProps<typeof defaultTag>
+
 export const Button = forwardRef<
   ElementRef<typeof defaultTag>,
-  ComponentProps<typeof defaultTag>
->((props, ref) => <button {...props} ref={ref} />);
+    ButtonProps
+>((props, ref) => <button style={{border:"none"}} {...props} ref={ref} />);
```

Comment:
so far things like that went into default style at the definition of the primitive  component

https://github.com/webstudio-is/webstudio-designer/blob/main/app/shared/canvas-components/primitives/box.tsx#L5

but we need to think carefully again about this, probably need to merge designer primitives with the SDK

Reply:
This should not be there. It was a debug code so a good catch!